### PR TITLE
Education-Translations (Turkish) (Trading Bot Layers Tutorial)

### PR DESCRIPTION
**Pull request by prepheadrus**
Docs-Nodes\T\Trading\Trading-Episode-Counters\trading-episode-counters.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-010-tutorial-step-1-2-3-counting.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-011-tutorial-step-trading-bot-layers-episode-counters-01.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-012-tutorial-step-how-about-the-rest.json
Docs-Nodes\E\Entry\Entry-Target-Rate\entry-target-rate.json
Docs-Nodes\E\Exit\Exit-Target-Rate\exit-target-rate.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-015-tutorial-step-elephant-what-elephant.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-016-tutorial-step-making-sense-of-objects.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-017-tutorial-step-what-about-the-position.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-018-tutorial-step-the-beauty-of-the-triangle.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-019-tutorial-step-why-3-position-layers.json
Docs-Nodes\E\Episode\Episode-Quoted-Asset\episode-quoted-asset.json
Docs-Nodes\M\Market\Market\market.json
Docs-Nodes\O\Order\Order-Base-Asset\order-base-asset.json
Docs-Nodes\O\Order\Order-Quoted-Asset\order-quoted-asset.json
Docs-Nodes\O\Orders\Orders\orders.json
Docs-Nodes\P\Position\Position-Base-Asset\position-base-asset.json
Docs-Nodes\P\Position\Position-Quoted-Asset\position-quoted-asset.json
Docs-Nodes\S\Stage\Stage-Base-Asset\stage-base-asset.json
Docs-Nodes\S\Stage\Stage-Quoted-Asset\stage-quoted-asset.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-020-tutorial-step-remember-flexibility.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-021-tutorial-step-phased-management.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-022-tutorial-step-orders-in-order.json
Docs-Nodes\P\Periods\Periods\periods.json
Docs-Nodes\P\Positions\Positions\positions.json
Docs-Nodes\S\Strategies\Strategies\strategies.json
Docs-Nodes\T\Trading\Trading-Current\trading-current.json
Docs-Nodes\T\Trading\Trading-Episode\trading-episode.json
Docs-Nodes\M\Manage\Manage-Stage\manage-stage.json
Docs-Topics\T\Trading\Trading-Systems\trading-systems-007-the-manage-stage.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-023-tutorial-step-stages-time.json
Docs-Nodes\T\Trading\Trading-Engine\trading-engine.json
Docs-Nodes\C\Close\Close-Stage\close-stage.json
Docs-Nodes\O\Open\Open-Stage\open-stage.json
Docs-Nodes\T\Trigger\Trigger-Stage\trigger-stage.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-024-tutorial-step-episode-x.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-025-tutorial-step-understanding-objects-data.json
Docs-Nodes\T\Trading\Trading-System\trading-system.json
Docs-Nodes\F\Formula\Formula\formula.json
Docs-Nodes\I\Initial\Initial-Targets\initial-targets.json
Docs-Nodes\S\Strategy\Strategy\strategy.json
Docs-Nodes\T\Trading\Trading-Strategy\trading-strategy.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-026-tutorial-step-daily-files-vs.-market-files.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-027-tutorial-step-bring-me-back-to-earth.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-028-tutorial-topic-trading-system-layers.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-029-tutorial-step-trading-system-errors.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-030-tutorial-step-trading-system-highlights.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-033-tutorial-step-trading-system-values.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-031-tutorial-step-trading-bot-layers-trading-system-highlights.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-032-tutorial-step-quick-refresher-on-stages.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-034-tutorial-step-the-rest-of-the-crew.json
Docs-Tutorials\T\Trading\Trading-Bot-Layers\trading-bot-layers-035-tutorial-step-time-to-go-out-and-play.json

### CODE REVIEW

**Review by gitforkgit — DISMISSED** [review]
Everything look fine to me. I couldn`t see any uncommonities. @harrellbm From my perspective this Pull request is able to merge. :)

**Comment by harrellbm** [comment]
@prepheadrus @gitforkgit With the PRs that got merged before this one there are now a few conflicts.  You should be able to use the forum post about conflicts to fixe them and then we can get this merged!

**Comment by gitforkgit** [comment]
@prepheadrus This should be a quick fix. Just click on Resolve conflicts and follow the short tutorial from @harrellbm: https://forum.superalgos.org/t/how-to-resolve-a-conflicting-translation/103?u=harrellbm

**Comment by prepheadrus** [comment]
> @prepheadrus This should be a quick fix. Just click on Resolve conflicts and follow the short tutorial from @harrellbm: https://forum.superalgos.org/t/how-to-resolve-a-conflicting-translation/103?u=harrellbm

hey @gitforkgit , i saw it and fixed it..thanks

**Comment by gitforkgit** [comment]
@prepheadrus Awesome, i'll look at the pull request again and mark it as mergable if everything fits :+1:

**Comment by prepheadrus** [comment]
> 👍

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 1]
```diff
@@ -40,6 +40,11 @@
                     "updated": 1638338966721
                 },
                 {
+                    "language": "TR",
+                    "text": "Bu hiyerarşiyi iki ana amaç için kullanacaksınız:",
+                    "updated": 1642504350982
+                },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:47]: The bracket before the spanish translation is not in line with the closing bracket from the Turkey one.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json [hunk 2]
```diff
@@ -113,6 +128,11 @@
                     "updated": 1638338139127
                 },
                 {
+                    "language": "TR",
+                    "text": "Bir ticaret sistemi birden fazla stratejiye sahip olduğunda, birinci strateji ikinciye, ikinci strateji üçüncüye göre önceliğe sahiptir ve bu böyle devam eder. Bu, stratejilerin bir sırayla değerlendirildiği anlamına gelir. Belirli bir strateji tetiklendiğinde, kuyrukta kalan stratejiler, strateji tetiklenene kadar artık değerlendirilmez. Başka bir deyişle, tek bir ticaret sistemi içinde birden fazla strateji konuşlandırıldığında, herhangi bir anda yalnızca bir strateji ticaret yapabilir ve ticaret sistemi düğümü etrafındaki sipariş tarafından öncelik verilir.",
+                    "updated": 1642513191527
+                     },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:135]: Brackets not in line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json [hunk 3]
```diff
@@ -71,6 +76,11 @@
                     "updated": 1640191081570
                 },
                 {
+                    "language": "TR",
+                    "text": "Superalgos Protokolünü takip eden stratejileri tanımlamak, stratejileri dört aşamaya ayırmak için bir ticaret sistemi kullanırsınız. Bu aşamalar  Trigger, Open, Manage, and Close aşamalarıdır..",
+                    "updated": 1642513118816
+                  },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:83]: Brackets not in line

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json [hunk 4]
```diff
@@ -134,6 +154,11 @@
                     "updated": 1638338156830
                 },
                 {
+                    "language": "TR",
+                    "text": "Yukarıdakilerin bir sonucu olarak, stratejilerin farklı pazarlarda işlemesini veya stratejilerin aynı anda pozisyon alabilmesini istiyorsanız, bu stratejileri farklı ticaret sistemlerinde kurmalısınız.",
+                    "updated": 1642513209866
+ },
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:160]: Brackets not in line.

**Review by gitforkgit — COMMENTED** [review]
I think it are only those very small and easy fixes. After you corrected them, I will go trough all files and then approve if nothing appears to be wrong to me.
Only a small step left and then we can merge :+1:

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json (hunk 4 above)
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: fix it.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json (hunk 3 above)
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: fix it

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json (hunk 2 above)
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: fix it

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json (hunk 1 above)
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json]: fix it

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json (hunk 4 above)
Comment (gitforkgit) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json]: :+1: I will review again

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 5]
```diff
@@ -116,9 +142,15 @@
                     "updated": 1638339266880
                 },
                 {
+                    "language": "TR",
+                    "text": "Aynı ticaret motoru, bir veri yapısının yalnızca düğüm tabanlı bir temsili olduğundan, birden fazla ticaret oturumunda kullanılabilir.",
+                    "updated": 1642504933300
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:149]: Those brackets need to be more left, maybe you can orient on already merged translations or files you also commited a bit more down. This occurs at some parts in this file.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 6]
```diff
@@ -133,9 +165,16 @@
                     "updated": 1638339437677
                 },
                 {
+
+                    "language": "TR",
+                    "text": "Bir Görev her çalıştırıldığında, Veri Madeni'ndeki (Data Mine) Ağ ve Bot tanımlarının bir anlık görüntüsü ile birlikte bu veri yapısının bir anlık görüntüsü UI'den Görev Sunucusuna (Task Manager) gönderilir ve oradan çalışan Bot tarafından tüketilir. Bu, ikinci bir Botun aynı Ticaret Motoru anlık görüntüsünü alabileceği ve diğer botların aynı bilgilerle beslenip beslenmediğini asla bilemeyeceği anlamına gelir.",
+                    "updated": 1642505103739
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:173]: Same here.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 7]
```diff
@@ -188,11 +243,18 @@
                     "updated": 1638339838848
                 },
                 {
+
+                    "language": "TR",
+                    "text": "Değiştirmek istediğiniz bir şey varsa, Ticaret Motoruna (Trading Engine) ticaret sisteminizden (Trading System) erişirsiniz. Kullanıcı Tanımlı Kod (User Defined Code) düğümünün tüm amacı budur.",
+                    "updated": 1642505339750
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:251]: same here

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json [hunk 8]
```diff
@@ -92,6 +102,11 @@
                     "updated": 1638338107057
                 },
                 {
+                    "language": "TR",
+                    "text": "Stratejileri aşamalar halinde tanımlama kavramı, her stratejiyi aynı çerçeve ile yürütmek için bir çerçeve sağladığından ve herhangi bir sayıda stratejiye dönüşebilecek ölçeklenebilir ticaret sistemlerinin geliştirilmesine katkıda bulunduğundan, ticaret sisteminin metodik yönü için esastır.",
+                    "updated": 1642513147547
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-System/trading-system.json:109]: Same here.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 9]
```diff
@@ -56,9 +61,15 @@
...
                 {
+                    "language": "TR",
+                    "text": "Ticaret sistemlerinizin ticaret botu tarafından işlenen bilgilere erişmesine izin vermek. Bu, stratejilerin, bot çalışırken verilen veya doldurulan siparişler gibi değiş tokuşla ilgili olanlar da dahil olmak üzere mevcut ve geçmiş olayları takip etmesine ve bunlara tepki vermesine olanak tanır.",
+                    "updated": 1642504373870
+                   },
+                   {
                     "language": "EL",
                     "text": "Για να επιτρέψετε στα συστήματα συναλλαγών σας να έχουν πρόσβαση στις πληροφορίες που επεξεργάζεται το ρομπότ συναλλαγών. Αυτό επιτρέπει στις στρατηγικές να παρακολουθούν και να αντιδρούν σε τρέχοντα και προηγούμενα γεγονότα — συμπεριλαμβανομένων εκείνων που αφορούν το ανταλλακτήριο, όπως εντολές που έχουν υποβληθεί ή εκπληρωθεί — καθώς δουλεύει το ρομπότ.",
                     "updated": 1641828420690
+
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:72]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 10]
```diff
@@ -99,6 +120,11 @@
                     "updated": 1638339236191
                 },
                 {
+                    "language": "TR",
+                    "text": "Bir Ticaret Motoru iki Ticaret Botu arasında paylaşılabilir mi?",
+                    "updated": 1642504459878
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:127]: Brackets must be more left.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 11]
```diff
@@ -116,9 +142,15 @@
...
                 {
+                    "language": "TR",
+                    "text": "Aynı ticaret motoru, bir veri yapısının yalnızca düğüm tabanlı bir temsili olduğundan, birden fazla ticaret oturumunda kullanılabilir.",
+                    "updated": 1642504933300
+                   },
+                   {
                     "language": "EL",
                     "text": "Η ίδια μηχανή συναλλαγών μπορεί να χρησιμοποιηθεί από πολλαπλές συνεδρίες συναλλαγών, επειδή είναι απλώς μια αναπαράσταση μιας δομής δεδομένων που βασίζεται σε κόμβους.",
                     "updated": 1641828487715
+
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:153]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 12]
```diff
@@ -160,9 +204,15 @@
                     "updated": 1638339666496
                 },
                 {
+                    "language": "TR",
+                    "text": "UI aracılığıyla ticaret motorunun içindeki hiçbir şeyi fiziksel olarak değiştirmeniz gerekmez. Ticaret motoru, ticaret sisteminin değerlerini depolamak için kullandığı veri yapısının görsel bir temsilidir.",
+                    "updated": 1642505145310
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:211]: Brackets must be more left.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 13]
```diff
@@ -160,9 +204,15 @@
...
                     "language": "EL",
                     "text": "Δεν χρειάζεται να αλλάξετε φυσικά τίποτα στη μηχανή συναλλαγών μέσω της UI (Διεπαφής Χρήστη). Η μηχανή συναλλαγών είναι μια οπτική αναπαράσταση της δομής δεδομένων που χρησιμοποιεί το σύστημα συναλλαγών για να αποθηκεύσει τις αξίες του, ας το πούμε έτσι.",
                     "updated": 1641828598544
+
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:215]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 14]
```diff
@@ -188,11 +243,18 @@
...
+                    "language": "TR",
+                    "text": "Değiştirmek istediğiniz bir şey varsa, Ticaret Motoruna (Trading Engine) ticaret sisteminizden (Trading System) erişirsiniz. Kullanıcı Tanımlı Kod (User Defined Code) düğümünün tüm amacı budur.",
+                    "updated": 1642505339750
+                   },
+                   {
                     "language": "EL",
                     "text": "Έχετε πρόσβαση στη Μηχανή Συναλλαγών μέσα από το σύστημα συναλλαγών σας, εάν υπάρχει κάτι εκεί που θέλετε να αλλάξετε. Αυτός είναι ολόκληρος ο σκοπός του κόμβου User Defined Code (Κώδικας που Ορίζεται από το Χρήστη).",
                     "updated": 1641828639843
+
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:255]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 15]
```diff
@@ -72,6 +83,11 @@
                     "updated": 1638339093835
                 },
                 {
+                    "language": "TR",
+                    "text": "Tasarım alanı görsel ortamı ve çizelgeler üzerindeki paneller aracılığıyla alım satım botunun eylemlerini takip etmek. Çalışma zamanı bilgilerini analiz ederek, bir alım satım seansı boyunca ne olduğu, ne zaman ve neden olduğu hakkında ayrıntılı bir anlayış elde edebilirsiniz.",
+                    "updated": 1642504426556
+                   },
+                   {
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:90]: Brackets must be more left.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 16]
```diff
@@ -133,9 +165,16 @@
...
+                    "text": "Bir Görev her çalıştırıldığında, Veri Madeni'ndeki (Data Mine) Ağ ve Bot tanımlarının bir anlık görüntüsü ile birlikte bu veri yapısının bir anlık görüntüsü UI'den Görev Sunucusuna (Task Manager) gönderilir ve oradan çalışan Bot tarafından tüketilir. Bu, ikinci bir Botun aynı Ticaret Motoru anlık görüntüsünü alabileceği ve diğer botların aynı bilgilerle beslenip beslenmediğini asla bilemeyeceği anlamına gelir.",
+                    "updated": 1642505103739
+                   },
+                   {
                     "language": "EL",
                     "text": "Κάθε φορά που εκτελείται μια Task (Εργασία), ένα στιγμιότυπο αυτής της δομής δεδομένων, μαζί με ένα στιγμιότυπο των ορισμών του Δικτύου και των Ρομπότ στο Data Mine (Εξορυκτήριο Δεδομένων) τους αποστέλλεται από τη UI (διεπαφή χρήστη) στον Task Server (Διακομιστή Εργασιών) και από εκεί χρησιμοποιείται από το Ρομπότ που εκτελείται. Αυτό σημαίνει ότι ένα δεύτερο Ρομπότ μπορεί να λάβει το ίδιο στιγμιότυπο της Μηχανής Συναλλαγών και δεν θα μάθει ποτέ εάν άλλα ρομπότ τροφοδοτήθηκαν με τις ίδιες πληροφορίες ή όχι.",
                     "updated": 1641828540851
+
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:177]: Blank line.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json [hunk 17]
```diff
@@ -188,11 +243,18 @@
                     "updated": 1638339838848
                 },
                 {
+
```
Comment (gitforkgit) [inline Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json:246]: Blank line.

**Review by gitforkgit — CHANGES_REQUESTED** [review]
Only few small adjustements left. Beside the Two Files all other 54 Files look great. Everywhere where i commented Blank line, you don`t need but can remove them. If you don`t see the brackets aligned wrong, let me know. Only a small step left before merging.

File: Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json (hunk 5 above)
Comment (prepheadrus) [reply Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json]: I did not fully understand. can you be a little more clear?

**Comment by gitforkgit** [comment]
@harrellbm can you have a look?

**Review by harrellbm — APPROVED** [review]

**Pull request merged**